ui/widgets/sidebar/src/components/sidebar-component.tsx

```diff
@@ -199,7 +198,9 @@ const SidebarComponent: React.FC<RootComponentProps> = props => {
 
   async function handleLogout() {
     await logoutQuery.mutateAsync();
-    // add other logic after logout
+    await queryClient.invalidateQueries({
+      queryKey: ['LOGIN_STATE'],
```

Comment:
export and use const `LOGIN_STATE_KEY` instead of redefining it here

Reply:
I forgot :). Will fix it